tests/ios/xcodeproj/BUILD.bazel

```diff
@@ -222,27 +221,3 @@ xcodeproj(
         "//tests/ios/frameworks/dynamic:App",
     ],
 )
-
-xcodeproj(
```

Comment:
Yeah the rules_ios workspace can call `load_xcahmmer_depenencies` and keep it included here but a customer can select to opt in if they need it still w/o removing the testing for it.

Reply:
I agree, but what if `load_xchammer_dependencies = False` (and not `True`) breaks and we don't notice because our CI is not checking that? Are you suggesting we wait for someone to report?

Reply:
>we don't notice because our CI is not checking that?
In the directory `tests` we can still run this on CI; but if you don't use XCHammer for say one of the apps then it won't download it for that app. This way we can keep ensuring it works

Reply:
Actually, I guess `load_xchammer_dependencies = True` imples `load_xchammer_dependencies = False` plus the additional XCHammer deps right? Maybe I'm wrong here and my concern is already addressed with even with `load_xchammer_dependencies = True` in `rules_ios`'s `WORKSPACE` file

Reply:
I'll bring this back then, if the above makes sense my concern with vetting `load_xchammer_dependencies = False` doesn't apply

Reply:
reverted removing the examples for the reasons above